app/models/topic_category.rb

```diff
@@ -0,0 +1,6 @@
+class TopicCategory < ApplicationRecord
+  belongs_to :community
+  has_many :topics, dependent: :nullify
+
+  validates :name, presence: true, uniqueness: { scope: :community_id }
```

Comment:
Remember to normalize string columns that are modified by user-input - it's possible to create a category with leading and trailing spaces now:

![Screenshot 2020-09-24 at 5 35 43 PM](https://user-images.githubusercontent.com/98546/94142786-703ace00-fe8c-11ea-8663-947a669aa5b8.png)